internal/repository/relationship_correction_inline_embedding.go

```
@@ -0,0 +1,222 @@
+		if err != nil {
+			return err
+		}
+		objectID, objectName, objectValueID, objectValue, objectUnit, err := correctionObjectProjection(ctx, tx, input.TeamID, source, projectionNames, objectPatch)
+		if err != nil {
+			return err
+		}
+		predicateKey := source.PredicateKey
+		if effective.Patch.Predicate != nil {
+			predicate, err := loadLatestActivePredicateDefinition(ctx, tx, input.TeamID, effective.Patch.Predicate.Key)
+			if err != nil {
+				return nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 103) but it returns nil

(nilerr)